gallery/src/components/NxForm/NxFormPage.tsx

```diff
@@ -247,6 +247,44 @@ const NxFormPage = () =>
           </NxTableRow>
         </NxTableBody>
       </NxTable>
+      <section className="nx-tile-subsection">
+        <header className="nx-tile-subsection__header">
+          <h3 className="nx-h3">NxForm Submit and Cancel Button Classes</h3>
+        </header>
+        <p className="nx-p">
+          In order to differentiate them from other buttons that might be on the page the
+          <code className="nx-code">NxForm</code> cancel and submit buttons have custom classes.
+        </p>
+        <NxTable className="nx-table--gallery-props">
+          <NxTableHead>
+            <NxTableRow>
+              <NxTableCell>Class</NxTableCell>
+              <NxTableCell>Location</NxTableCell>
+              <NxTableCell>Details</NxTableCell>
+            </NxTableRow>
+          </NxTableHead>
+          <NxTableBody>
+            <NxTableRow>
+              <NxTableCell><code className="nx-code">nx-btn--form-cancel</code></NxTableCell>
+              <NxTableCell>
+                Applied to the default <code className="nx-code">NxForm</code> Cancel button.
+              </NxTableCell>
+              <NxTableCell>
+                A CSS class that can be used to identify the Cancel button on the page for testing or other purposes.
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell><code className="nx-code">nx-btn--form-submit</code></NxTableCell>
```

Comment:
Need to update these

Reply:
> Need to update these

Right: https://github.com/sonatype/sonatype-react-shared-components/pull/320/commits/6c9a620d88fa6f452c18fa00a2e12aaa4e742255